rerun_py/rerun_sdk/rerun/blueprint/api.py

```diff
@@ -418,12 +418,16 @@ def __init__(
         self.auto_space_views = auto_space_views
         self.auto_layout = auto_layout
 
-        # If there's no `root_container`, switch `auto_layout`` and `auto_space_views`` defaults to `True`.
-        if not hasattr(self, "root_container"):
+        if len(contents) == 0:
+            # If there's no content, switch `auto_layout`` and `auto_space_views`` defaults to `True`.
```

Comment:
The syntax highlighting in my brain wasn't working

```suggestion
            # If there's no content, switch `auto_layout` and `auto_space_views` defaults to `True`.
```